Yoast/wordpress-seo
# Add dashboard wincher connect

## Context

Display the Wincher performance report component on the dashboard even for not logged-in users, and add a reconnect button and fake table data.

## Summary

This PR can be summarized in the following changelog entry:

* Display the Wincher performance report component for not logged-in users, on the dashboard.
* Add connect button.
* Add fake table data.
* Add table description.

## Test instructions
* Go to the dashboard.
* Check "Top performing keyphrases on your site" section (i.e. Wincher performance component)
 
### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* With a not connected user, the Wincher performance component should be displayed.
* Connect button should invoke login action

#### Relevant test scenarios

### Test instructions for QA when the code is in the RC





## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

* Wincher performance report component should be able to display data without regression for logged-in users.
* Edge cases (e.g. unauthorized to fetch table data, empty data, etc) handling should work without regression.

## UI changes


## Other environments


## Documentation


## Quality assurance


## Innovation


Fixes #
